.goreleaser.yml

```
@@ -0,0 +1,58 @@
+before:
+  hooks:
+  - go mod download
+builds:
+- id: git-ghost
+  binary: git-ghost
+  env:
+  - CGO_ENABLED=0
+  - GO_VERSION=1.19
+  goos:
+  - linux
+  - darwin
+  goarch:
+  - amd64
+  - arm64
+  ldflags:
+  - -s -w -X main.Version={{.Version}}
```

`Version, Revision` vars live in `github.com/pfnet-research/git-ghost/cmd` package.

```suggestion
  - -s -w -X github.com/pfnet-research/git-ghost/cmd.Version.Version={{.Version}} -X github.com/pfnet-research/git-ghost/cmd.Revision={{.ShortCommit}}
```